README.md

```diff
@@ -133,27 +133,65 @@ $config = [
 Combining the previous steps will yield the following result:
 
 ```php
-add_filter('MyProject/ACF/fields/field_1', function ($field) {
+add_filter('MyProject/ACF/fields/field1', function ($field) {
   return [
     'label' => 'Sub Title',
-    'name' => 'sub_title',
+    'name' => 'subTitle',
     'type' => 'text'
-  ]
+  ];
 });
 
 add_filter('MyProject/ACF/locations/postTypePost', function ($location) {
   return [
     'param' => 'post_type',
     'operator' => '==',
     'value' => 'post'
+  ];
+});
+
+$config = [
+  'name' => 'group1',
+  'title' => 'My Group',
+  'fields' => [
+    'MyProject/ACF/fields/field1'
+  ],
+  'location' => [
+    [
+      'MyProject/ACF/locations/postTypePost'
+    ]
   ]
+];
+
+ACFComposer\ACFComposer::registerFieldGroup($config);
+```
+
+Executing this code will add a field with the **name** `subTitle` to all posts. The **key** will be a combination of the field group name, potentially all parent field names, and the field's name itself. In this case, that is `field_group1_subTitle`.
+
+### Filter arguments
+
+There is another caveat when working with reusable components in ACF. While the flexible content field from ACF Pro gives you everything you need for adding multiple components of the same type to one field group, this is not possible for regular field groups.
+
+For example, you define a set of fields for a simple WYSIWYG component:
```

Comment:
I would split the example on the explanation into two clear blocks, rather than one 'run-on' sentence + explanation.

"For example, if you define a set of fields for a simple WYSIWYG component and then want to add it to a field group multiple times, the result will be two Wysiwyg components being displayed, but each one would have the same name, and thus overwrite each other's data.

[ example code block here ]"